catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet

```diff
@@ -0,0 +1,269 @@
+    United States employed mainly images from 2019, while Alaska land cover maps are mainly based on the 
+    use of images from 2021. The land cover map for Mexico was based on land cover change detection 
+    between 2015 and 2020 Mexico Landsat 8 mosaics.
+
+    In order to generate a seamless and consistent land cover map of North America, national maps were 
+    generated for Canada by the CCRS; for Mexico by CONABIO, INEGI, and CONAFOR; and for the United States 
+    by the USGS. Each country chose their own approaches, ancillary data, and land cover mapping methodologies
+    to create national datasets. This North America dataset was produced by combining the national land cover datasets. 
+    The integration of the three national products merged four Land Cover map sections, Alaska, Canada, 
+    the conterminous United States and Mexico.
+
+    See also:
+
+    * Natural Resources Canada has North American Land Cover information available online at 
+      (https://open.canada.ca/data/en/dataset/ee1580ab-a23d-4f86-a09b-79763677eb47)
+ 
+    * The National Commission for the Knowledge and Use of Biodiversity has North American Land Cover 
+    information available online at (https://www.biodiversidad.gob.mx/monitoreo/cobertura-suelo)
+
+    * The U.S. Geological Survey has North American Land Cover information available online at (www.mrlc.gov)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'land_surface',
+    'land_use',
+    'land_cover',
+    'reflectance',
+    'landsat',
+    'remote_sensing',
+    'north_america',
+  ],
+  providers: [
+    ee.producer_provider(
+      'The Commission for Environmental Cooperation (CEC)',
+      'http://www.cec.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      175, 14, -50, 84,
+      '2019-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'year',
+    interval: 1,
+  },
+  summaries: {
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'landcover',
+        description: |||'Land cover classification grid cell value. Values are categorized into the following: 1 = Temperate or sub-polar needleleaf forest, 2 = Sub-polar taiga needleleaf forest, 3 = Tropical or sub-tropical broadleaf evergreen forest, 4 = Tropical or sub-tropical broadleaf deciduous forest, 5 = Temperate or sub-polar broadleaf deciduous forest, 6 = Mixed forest, 7 = Tropical or sub-tropical shrubland, 8 = Temperate or sub-polar shrubland, 9 = Tropical or sub-tropical grassland, 10 = Temperate or sub-polar grassland, 11 = Sub-polar or polar shrubland-lichen-moss, 12 = Sub-polar or polar grassland-lichen-moss, 13 = Sub-polar or polar barren-lichen-moss, 14 = Wetland, 15 = Cropland, 16 = Barren lands, 17 = Urban and built-up, 18 = Water, 19 = Snow and ice.'|||,
+        'gee:units': units_class,
```

Comment:
Comment out the units_class. We need to convert this to a gee:classes.

e.g. https://github.com/google/earthengine-catalog/blob/main/catalog/USGS/USGS_NLCD.jsonnet#L144

But we can do that as a follow on PR, so just make a TODO